infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -234,7 +234,11 @@ private void prefixTrim(CorfuPayloadMsg<TrimRequest> msg, ChannelHandlerContext
                             IServerRouter r) {
         try {
             TrimRequest req = msg.getPayload();
-            batchWriter.prefixTrim(req.getAddress());
+            // Here the token is not used as is to perform the trim as the epoch at which the checkpoint was completed
```

Comment:
why not to transform this doc to javadoc for this method?

Reply:
Done.